react_app/src/Form.js

```diff
@@ -10,14 +10,27 @@ class EmbeddingForm extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      values: [],
+      selectedValue: "",
       createMode: false
     };
 
+    this.setSelection = this.setSelection.bind(this);
```

Comment:
Just discovered this. Looks like some people use this style but it isn't that readable to me. Can you do:
this.setSelectionFn_ = this.setSelection.bind(this)
Instead? and use setSelectionFn?
@askewc WDYT?

Reply:
Fixed!